src/Components/EnvironmentPicker/EnvironmentPicker.types.ts

```diff
@@ -54,3 +54,35 @@ export type StorageAccountToContainersMapping = {
     storageAccountUrl: string;
     containerNames: Array<string>;
 };
+
+// START of Actions
+export const SET_ENVIRONMENT_ITEMS = 'SET_ENVIRONMENT_ITEMS';
```

Comment:
there should be a mapping of action to payload type definitions. Please look at the SceneViewContext, DeeplinkContext or SceneDialog.state for examples.

Reply:
yep changed this as a part of fix for your other comment: https://github.com/microsoft/iot-cardboard-js/commit/0892e957c7ccd22f1286e50bf5f6e6245b384c8d#diff-339ca7afc846835ed41cf981fb15ec5df52a13b302731c5e1e8c6f67fed087eeR89